external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaRelation.scala

```diff
@@ -143,4 +143,6 @@ private[kafka010] class KafkaRelation(
         validateTopicPartitions(partitions, partitionOffsets)
     }
   }
+
+  override def toString: String = "kafka"
 }
```

Comment:
How about this? Just show the most important parameters like `s"KafkaRelation(strategy=$strategy, start=$startingOffsets, end=$endingOffsets)"`?